packages/circle-demo-webapp/app/lib/types.ts

```diff
@@ -760,3 +760,19 @@ export interface EventLog {
   txHash: string;
   userOpHash: string;
 }
+
+export interface ErrorResponse {
+  error: {
+    code: number;
+    message: string;
+    errors: {
+      message: string;
+    }[];
+  };
+}
+
+export interface ErrorResponseObject {
```

Comment:
@avkos this type was moved to packages/circle-demo-webapp/app/lib/server.responses.ts and were renamed to CircleErrorResponse.
use successResponse and errorResponse in the future for consistency